pkg/apis/awsprovider/v1alpha1/types.go

```diff
@@ -237,6 +238,9 @@ var (
 
 	// SecurityGroupControlPlane defines a Kubernetes control plane node role
 	SecurityGroupControlPlane = SecurityGroupRole("controlplane")
+
+	// SecurityGroupLB defines a container for the cloud provider to inject its load balancer ingress rules
+	SecurityGroupLB = SecurityGroupRole("lb")
```

Comment:
suggest creating package constants for `lb`, `controlplane`, `node` and `bastion`. I'll happy to take that as a follow-up.

Reply:
I started down this path, but I think it's large enough and self-contained enough to make more sense as a separate PR